### ISSUE TITLE
Support non-master default branch

### ISSUE THREAD

**Issue by yo-ga** [spenserblack/actions-wiki#74]
For now, the script pull `master` branch and work on it. However, the GitHub has changed the default branch for new repositories from `master` to `main`. Kindly add the optional parameters for choosing the branch.

**Comment by spenserblack** [comment]
Yeah, I was considering this input, but AFAIK all *wiki* default branches are still `master`.
